spaces/S000171/properties/P000057.md

```diff
@@ -0,0 +1,9 @@
+---
+space: S000171
+property: P000057
+value: false
```

Comment:
Instead of saying the space is not countable, can we assert P65 (continuum-sized) as it is more informative/precise in this case?

Reply:
Yep, I've only skimmed the source to meet the request in #130, so thanks for reading it more carefully to improve this contribution.